src/pybind/mgr/dashboard/frontend/src/app/shared/components/empty-state/empty-state.component.html

```diff
@@ -0,0 +1,20 @@
+<div class="container h-75">
+  <div class="row h-100 justify-content-center align-items-center">
+    <div class="blank-page">
+      <i class="mx-auto d-block"
+         [ngClass]="[icons.large5x, icons.wrench]"></i>
+
+      <div class="mt-4 text-center">
+        <h4 i18n>{{ msg }}</h4>
+        <button class="btn btn-accent mt-3"
+                (click)="submitAction()"
+                name="empty-state-submit"
+                [attr.aria-label]="btnText"
+                i18n>{{ btnText }}</button>
+        <cd-helper *ngIf="helperText">
+        <span i18n>{{ helperText }}</span>
+        </cd-helper>
```

Comment:
BTW, rather than a separate info tooltip, I was just thinking on the [built-in tooltip](https://www.w3schools.com/tags/att_global_title.asp) of the button (just adding a `title="sample tooltip"` to the `button` element will result in a tooltip).

Reply:
I misunderstood it. thanks.